readme.md

```diff
@@ -102,24 +99,22 @@ mbo.finalize().then(function(mediaBlobs) {
 ```
 
 ### Split Method
-The split method allows the author to split a *blob* into two separate MediaBlobs at a given time.
+The split method allows the author to split a *blob* into two separate MediaBlobs at a given time. Due to the nature of this operation, it should be the last
```

Comment:
Is there a way to enforce the that the spit operation should be last by making it impossible for the developer to call split before other operations?

For example, could we move split() to the MediaBlob interface?  Or should we add a splitAndFinalize() method to the MediaBlob operation?

Reply:
The proposed error checking on finalize() makes sure that split() if called is always the last operation called before finalize().

Do we still need to add more guardrails here ?

Reply:
Maybe we should add more guardrails if possible.  It's more developer friendly to create an API that cannot be used incorrectly.  What advantages do we gain by tying split() to MediaBlobOperation?

Reply:
I agree. I will move this to a new issue.